Api Web/User.cs

```diff
@@ -18,6 +19,13 @@ public class User
 
         public string Email { get; set; }
 
+        public int EmployeeId { get; set; }
+
         public DateTime DateOfBirth { get; set; }
+
+        //foreing key for Departments
+        public int DepartmentEmployeeId { get; set; }
+
+        public Department Departments { get; set; }
```

Comment:
Same here, Department != Departments, we must be consistent with names that make sense.